src/bridge/command.rs

```diff
@@ -89,19 +90,29 @@ fn neovim_ok(bin: &str, args: &[String]) -> Result<bool> {
             // The output is not utf8 on Windows and can contain special characters.
             // But a lossy conversion is OK for our purposes
             let stdout = String::from_utf8_lossy(&output.stdout);
+
             if !(stdout.starts_with("NVIM v") && output.stderr.is_empty()) {
+                let win_wsl_screen_size_error =
+                    Regex::new(r"your \d+x\d+ screen size is bogus. expect trouble").unwrap();
+                let stderr = String::from_utf8_lossy(&output.stderr);
+                for line in stderr.lines() {
+                    if win_wsl_screen_size_error.is_match(line) {
+                        return Ok(true);
```

Comment:
Pushed a potential resolution that partitions matching and non-matching lines, then prints the full error if there are lines that are non-matching. 

This way when there are extra errors, we don't print the error about the bogus screen size to the user since that would be misleading. We only serve them the errors they should care about. 

```rust
let (matching_lines, non_matching_lines): (Vec<_>, Vec<_>) = stderr
  .lines()
  .partition(|line| win_wsl_screen_size_error.is_match(line));
if matching_lines.len() == stderr.lines().count() {
  return Ok(true);
}

let error_message_prefix = format!(
  concat!(
    "ERROR: Unexpected output from neovim binary:\n",
    "\t{bin} -v\n",
    "stdout: {stdout}\n",
    "stderr: {err}\n",
    "Check that your shell doesn't output anything extra when running:",
    "\n\t"
  ),
  bin = bin,
  stdout = stdout,
  err = non_matching_lines.join("\n"),
);

```